pkg/Library/App/Nodejs/App.js

```diff
@@ -37,7 +37,7 @@ export const App = class {
     });
     //await loadCss(`${this.paths.$library ?? '.'}/Dom/Material/material-icon-font/icons.css`);
     // TODO(sjmiles): pick a syntax
-    const recipes = [DevToolsRecipe, ...(this.recipes ?? this.recipes ?? [])];
+    const recipes = [DevToolsNode, ...(this.recipes ?? this.recipes ?? [])];
```

Comment:
It says _recipes_.

Reply:
i think this file is outdated, we should no longer have addRecipes API... or do we want to keep them both? i'm confused 😕

Reply:
You are right, this file is outdated. But IMO we should either fix it all the way, discard it, or leave it alone. Generally the notion of converting Recipes to Graphs is not wrong, it's just I think it's ok if we do it kind of slowly.